workspace.json

```diff
@@ -9569,6 +9569,59 @@
           }
         }
       }
+    },
+    "services-contentful-entry-tagger": {
+      "root": "apps/services/contentful-entry-tagger",
+      "sourceRoot": "apps/services/contentful-entry-tagger/src",
+      "projectType": "application",
+      "prefix": "services-contentful-entry-tagger",
+      "architect": {
+        "build": {
+          "builder": "@nrwl/node:build",
+          "outputs": ["{options.outputPath}"],
+          "options": {
+            "outputPath": "dist/apps/services/contentful-entry-tagger",
+            "main": "apps/services/contentful-entry-tagger/src/main.ts",
+            "tsConfig": "apps/services/contentful-entry-tagger/tsconfig.app.json",
+            "assets": ["apps/services/contentful-entry-tagger/src/assets"]
+          },
+          "configurations": {
+            "production": {
+              "optimization": true,
+              "extractLicenses": true,
+              "inspect": false,
+              "fileReplacements": [
+                {
+                  "replace": "apps/services/contentful-entry-tagger/src/environments/environment.ts",
+                  "with": "apps/services/contentful-entry-tagger/src/environments/environment.prod.ts"
+                }
```

Comment:
We stopped using this method of configuration.

Reply:
Should I remove the replace and with then? This got added automatically when I bootstrapped the service

Reply:
Just remove this to begin with. There is a library that is used to retrieve config if I recall correctly.

Reply:
Done :)